src/components/app/constants.js

```diff
@@ -28,12 +28,14 @@ export const PATHS = {
     CONTACT: '/contact',
     HOME: '/',
     MARKET: '/market',
+    PRODUCER_DATA_PAGE: '/producerData',
     PROFILE: '/profile',
     SIGN_IN: '/signin',
     SIGN_UP: '/signup',
 };
 
 export const USER_AVATAR = {
+    ADD_ITEM: "Add Item",
```

Comment:
this variable is not part of `USER_AVATAR` move it out of this

Reply:
OK

Reply:
This variable hasn't been moved

Reply:
Hmmm. I did move it. Something is definitely wrong somewhere